scripts/screens/LeaderDenScreen.py

```diff
@@ -268,21 +264,21 @@ def screen_switches(self):
 
         # if no one is alive, give a special notice
         if not get_living_clan_cat_count(Cat):
-            self.no_gathering = True
+            self.no_leader = True
             self.screen_elements["clan_notice_text"].set_text(
-                f" No one is left to attend a Gathering. "
+                " No one is left to attend a Gathering. "
             )
             self.screen_elements["outsider_notice_text"].set_text(
-                f" Outsiders do not concern themselves with a dead Clan. "
+                " Outsiders do not concern themselves with a dead Clan. "
             )
         # if leader is dead and no one new is leading, give special notice
         elif self.no_leader or game.clan.leader.dead or game.clan.leader.exiled:
-            self.no_gathering = True
+            self.no_leader = True
             self.screen_elements["clan_notice_text"].set_text(
-                f" Without no one to lead, the Clan can't focus on what to say at the Gathering. "
+                " Without no one to lead, the Clan can't focus on what to say at the Gathering. "
```

Comment:
```suggestion
                " With no one to lead, the Clan can't focus on what to say at the Gathering. "
```
might as well fix this typo while we're here